memory/table.go

```diff
@@ -136,6 +136,27 @@ func NewPartitionedTableWithCollation(name string, schema sql.PrimaryKeySchema,
 		}
 	}
 
+	newSchema := make(sql.Schema, len(schema.Schema))
+	for i, c := range schema.Schema {
+		cCopy := c.Copy()
+		if cCopy.Default != nil {
+			newDef, _, _ := transform.Expr(cCopy.Default, func(e sql.Expression) (sql.Expression, transform.TreeIdentity, error) {
+				switch e := e.(type) {
+				case *expression.GetField:
+					return expression.NewGetField(e.Index(), e.Type(), e.Name(), e.IsNullable()), transform.NewTree, nil
```

Comment:
What is the purpose of this line? It looks like it just creates a new getField identical to the old one.

Reply:
We have to serialize defaults without the table qualifier. I will add a comment